rs/tests/src/rosetta_tests/setup.rs

```diff
@@ -294,11 +294,11 @@ echo \"Rosetta container started \"
         .unwrap();
 
     // Add Rosetta image to config dir.
-    let path = get_dependency_path("rs/rosetta-api/")
+    let path = get_dependency_path("rs/rosetta-api")
         .into_os_string()
         .into_string()
         .unwrap();
-    let path = format!("{}/rosetta_image.tar", path);
+    let path = format!("{path}/rosetta_image.tar");
```

Comment:
Something like `path.join("rosetta_image.tar")` would be nicer but then path has to be a `Path`>